pkg/indexer/rpc_streamer/rpc_log_streamer.go

```diff
@@ -185,6 +186,19 @@ backfillLoop:
 			logger.Error("Context cancelled, stopping watcher")
 			return
 
+		case err, open := <-sub.Err():
+			if !open {
+				logger.Error("subscription channel closed, closing watcher")
+				return
+			}
```

Comment:
when doing an initial backfill, the sub cancellation might happen quite a bit while we are still backfilling. We should totally do the right thing, but tearing down everything seems like overkill.